docusaurus/docs/reactnative/customization/native-handlers.mdx

```diff
@@ -62,6 +62,14 @@ A function that deletes a file at a given local `uri`.
 | ------------------------------------------------------------------------------- | -------------------------------------------------------------------------- |
 | [`react-native-blob-util`](https://github.com/RonRadtke/react-native-blob-util) | [`expo-file-system`](https://docs.expo.io/versions/latest/sdk/filesystem/) |
 
+### `FlatList`
+
+A FlatList component, on Expo the standard React Native component is used, on React Native a modified FlatList better fit for two directional scrolling is used.
+
+| React Native CLI                                                           |
+| -------------------------------------------------------------------------- |
+| [`@stream-io/flat-list-mvcp`](https://github.com/GetStream/flat-list-mvcp) |
+
```

Comment:
do we stll needc this? confusing

Reply:
Well I am unsure if we need to add it here or not. Technically users can customize it since we have it available.

Reply:
lets remove the whole thing, lets make the mvcp lib to be deprecated also

Reply:
inside the SDK we use in 0.71 and less if installed... else we dont thats it

Reply:
But, we never used it for expo. I have removed the docs from here. Also, we can mention in v6 release docs, that if you are above 0.71 then you don't need to install the package and rest all is catered IMO. And about deprecating, I think I won't do unless we make v6 default.